website/docs/r/nginx_deployment.html.markdown

```diff
@@ -163,9 +163,11 @@ In addition to the Arguments listed above - the following Attributes are exporte
 
 * `id` - The ID of the NGINX Deployment.
 
-* `ip_address` - The IP address of the deployment.
+* `ip_address` - The IP address of the NGINX Deployment.
 
-* `nginx_version` - The version of deployed NGINX.
+* `nginx_version` - The version of the NGINX Deployment.
+
+* `dataplane_api_endpoint` - The dataplane API endpoint of the NGINX Deployment.
```

Comment:
Not important or a now problem, just mentioning it for future PRs but these should be listed in alphabetical order

Reply:
got it 👍

Reply:
quick question on this. What if certain properties are more important than the ones when listed alphabetically?

As a user of the NGINXaaS instance, I will care more about the IP address and the version of the instance rather than the dataplane endpoint.

Reply:
We don't tend to consider ordering via importance of properties since this differs per resource and can also be subjective. 

Properties that we consider important and are exempt from the alphabetical ordering that we try to follow and are always listed at the top are ones that are used to construct the resource's identifier and location.